index.qmd

```diff
@@ -0,0 +1,1154 @@
+---
+title: "Project oriented workflows"
+author: "Max Kuhn, E. David Aja, Shannon Pileggi"
+footer:  "[Project oriented workflows]()"
+logo: "img/wtf-logo-square-transparent.png"
+format: 
+  revealjs: 
+    height: 900
+    width: 1600
+    theme: [night, slides.scss]
+    highlight-style: a11y
+    transition: fade
+    slide-number: true
+editor: visual
+execute:
+  freeze: auto
+---
+
+# Getting started
+
+```{r}
+library(countdown)
+```
+
+```{r font-awesome-color}
+# fill for font awesome icons
+fa_fill <- "#C7B41D"
+```
+
+## Licensing
+
+<br>
+
+This work is licensed under a [Creative Commons Attribution-ShareAlike 4.0 International License](https://creativecommons.org/licenses/by-sa/4.0/) (CC BY-SA4.0).
+
+<br>
+
+  - Part 1: Original sides were authored by [**Shannon Pileggi**](https://www.pipinghotdata.com/){target="_blank"}, some added or altered by [**Max Kuhn**](https://max-kuhn.org/){target="_blank"}
+
+- Part 2: [**Max Kuhn**](https://max-kuhn.org/){target="_blank"}  largely created these
+
+- Part 3: Created from [**E. David Aja**](https://edavidaja.com/){target="_blank"}'s previous work
+
+Other authors are identified on a slide-by-slide basis. 
+
+None of this would exist without vision (and patience) of [**Jennifer Bryan**](https://jennybryan.org/){target="_blank"}
+
+## Checklist
+
+<br>
+
+`r fontawesome::fa("check", fill = fa_fill)` R installed? Pretty recent?
+
+     Recommended R ≥ 4.3.0
+
+`r fontawesome::fa("check", fill = fa_fill)` RStudio installed?
+
+     I'm on 2023.06.1 Build 524     
+
+`r fontawesome::fa("check", fill = fa_fill)` Packages?
+
+`install.packages(c("tidyverse", "rmarkdown", "usethis", "fs", "here"))`
+
+## Resources
+
+<br>
+
+Book: [What They Forgot to Teach You About R](https://rstats.wtf/){target="_blank"}
+
+<br>
+
+Book: [Happy Git and GitHub for the useR](https://happygitwithr.com/){target="_blank"}
+
+<br> 
+
+Website: [Tidyteam code review principles](https://code-review.tidyverse.org/){target="_blank"}
```

Comment:
Maybe not too relevant but I added it in case they are collaborating. Good stuff.